spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/actions/RewritePositionDeleteFilesSparkAction.java

```diff
@@ -133,16 +137,23 @@ public RewritePositionDeleteFiles.Result execute() {
   }
 
   private StructLikeMap<List<List<PositionDeletesScanTask>>> planFileGroups() {
-    CloseableIterable<PositionDeletesScanTask> fileTasks = planFiles();
+    Table deletesTable =
+        MetadataTableUtils.createMetadataTableInstance(table, MetadataTableType.POSITION_DELETES);
+    PositionDeletesTable.PositionDeletesBatchScan deletesScan =
```

Comment:
Can we do direct import to shorten the lines like in the sample snippet I mentioned above?